api/src/main/java/org/openmrs/User.java

```diff
@@ -93,6 +95,18 @@ public User(Person person) {
 		this.person = person;
 	}
 	
+	/*	public User(HttpSession sessionId) {
+			this.sessionId = sessionId.getId();
+		}*/
+	
+	public String getSessionId() {
```

Comment:
We really shouldn't be adding these fields here:

1. They shouldn't be persisted to the data base
2. A user can have more than one session
3. I don't think they are used anyway.